packages/flutter/lib/src/rendering/view.dart

```diff
@@ -247,6 +247,13 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     }
   }
 
+  /// Sends the provided [SemanticsUpdate] to the [FlutterView] associated with this [RenderView].
```

Comment:
nitty-nit: (here and elsewhere) we try to keep doc comments to ~80 char line length. Maybe you can reflow these?